.travis.yml

```diff
@@ -21,6 +23,8 @@ matrix:
       gemfile: gemfiles/i18n_0.3.gemfile
     - rvm: 2.2
       gemfile: gemfiles/i18n_0.3.gemfile
+    - rvm: 2.3.1
+      gemfile: gemfiles/i18n_0.3.gemfile
```

Comment:
Maybe it's just time to drop I18n 0.3 (it's not working with any Ruby version except 1.9.3)